src/components/constants.ts

```diff
@@ -34,7 +34,12 @@ export const palette = {
 };
 
 // Size
-export const DEFAULT_ICON_SIZE = { SMALL: 24, LARGE: 46 };
+export const DEFAULT_ICON_SIZE = {
+  SMALL: 23,
+  MEDIUM: 38,
+  LARGE: 40,
+  HUGE: 46,
+};
```

Comment:
Its funny that we call it "default icon size" when we have 4 variants of it :D

I get that it has been expanded one by one, but maybe it is time to call it "icon_sizes" =D

Reply:
True! 😃 I'll rename it!

Reply:
Done!